charts/opentelemetry-ebpf/values.yaml

```diff
@@ -0,0 +1,275 @@
+# Default values for opentelemetry-ebpf.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+nameOverride: ""
+fullnameOverride: ""
+
+clusterName: ""
+
+images:
+  tag: "v0.10.0"
```

Comment:
tag should use `""` for the default value and the chart should default to the appVersion if tag is not set.